code/sql.c

```diff
@@ -30,7 +30,7 @@ bool CSQLInterface::StartSQLServer(const char* host, const char* db, const char*
 
 	if(!mysql_real_connect(connection, host, user, pass, db, 0, nullptr, 0))
 	{
-		RS.Bug("Unable to connect to mysql database: %s\r\n", mysql_error(connection));
+		RS.Logger.Warn("Unable to connect to mysql database: {}\r\n", mysql_error(connection));
```

Comment:
Would this be better as an error as well?

Reply:
Fixed in latest commit.